test/jest-api.global-setup.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Knex = require('knex');
+
+const databases = {
+  postgres: {
+    client: 'postgres',
+    connection: {
+      host: '127.0.0.1',
+      port: 5432,
+      database: 'strapi_test',
+      user: 'strapi',
+      password: 'strapi',
+    },
+  },
+  mysql: {
+    client: 'mysql',
+    connection: {
+      host: '127.0.0.1',
+      port: 3306,
+      database: 'strapi_test',
+      user: 'root',
+      password: 'strapi',
+    },
+  },
+  sqlite: {
+    client: 'sqlite',
+    connection: {
+      filename: './tmp/data.db',
+    },
+    useNullAsDefault: true,
+  },
+};
+
+async function main(uuid) {
+  const knex = Knex(databases[process.env.DATABASE]);
+  // Lets create our database if it does not exist
+  await knex.raw(`CREATE DATABASE ${uuid};`);
+}
+
+async function CreateDB(uuid) {
```

Comment:
just for convention this should be lower case `createDB`

Reply:
also, super minor suggestion, maybe the main method could be inside this createDB ? as there are only two lines of code. wdyt?